From 50a71462a3291f7e08fa25f67b274792bb419211 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Tue, 14 Jul 2015 21:24:37 +0200
Subject: [PATCH 1/2] log2asc: include sys/time.h

Without this patch compilation against musl C library
breaks, because struct timeval cannot be found.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 log2asc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/log2asc.c b/log2asc.c
index 29d13e8..8fb88a9 100644
--- a/log2asc.c
+++ b/log2asc.c
@@ -48,6 +48,7 @@
 #include <unistd.h>
 
 #include <net/if.h>
+#include <sys/time.h>
 #include <linux/can.h>
 
 #include "lib.h"
-- 
2.1.4

